.github/workflows/tests.yml

```diff
@@ -0,0 +1,53 @@
+name: Python tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        include:
+          - python-version: "3.7"
+            django-version: "3.2"
+          - python-version: "3.9"
+            django-version: "3.2"
+          - python-version: "3.9"
```

Comment:
Python version 3.9, Django 3.2 is listed twice. Did you mean Python 3.8?

Reply:
indeed, opened a pr that fixes that and drop django 4.0